### DISCUSSION TITLE
有没有人也觉得小狼毫比fcitx5-rime反应要慢一些？

### DISCUSSION THREAD

**Discussion by piiiiq** [rime/home#1650] [Q&A]
我在同一台电脑上分别在Linux下使用fcitx5-rime和Windows下使用小狼毫（0.16.1版本）打字，同样的配置文件，明显感觉fcitx5-rime要快，fcitx5-rime从来没有卡顿过。而用小狼毫总感觉选词框弹出的速度比前者慢一些，小狼毫有时还会卡上半秒钟才弹出选词框。而且我在Linux上将CPU限制在最低频率，fcitx5-rime依然流畅，在Windows里不限制CPU频率，并将小狼毫进程优先级设为实时，依然感觉不如fcitx5-rime快，也依然会出现卡顿现象。

**Comment by piiiiq** [comment]
配置文件就是雾凇拼音，我用的是其中的微软双拼，我发了两遍吗？我不记得曾经发过一次了，所以又发了一次。


________________________________

**Comment by piiiiq** [comment]
使用小狼毫我为了尽可能地提高选词框的弹出速度，将选词框的阴影和圆角都关了，每页候选词数量5个，但是测试依然不如fcitx5-rime流畅，有时会出现明显的卡顿，选词框会延时半秒到一秒才能显示出来。

________________________________

**Comment by fxliang** [comment]
试下用原装的明月拼音，不修改样式配置的时候，再看下表现吧

**Reply by piiiiq** [reply]
算了，就这样吧。用了一段时间，也算适应了。怎么关闭讨论啊？我找不到关闭讨论的方法。
